_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -22,28 +22,28 @@ acknowledgments:
 
 ## Applicability
 
-Any [WAI-ARIA 1.1 state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is not empty (""), and that is specified on an HTML or SVG element.
+The rule applies to any HTML or SVG element that has any not empty (`""`) [WAI-ARIA 1.1 state or property][] specified.
```

Comment:
We actually should have examples (Passed and Failed) with SVG element, since they are applicable.

Reply:
I see your point. I have updated the applicability so it relies still on the state or property.